src/autoscaler/api/broker/broker.go

```diff
@@ -507,7 +508,13 @@ func (b *Broker) Bind(ctx context.Context, instanceID string, bindingID string,
 
 	policy, err := b.getPolicyFromJsonRawMessage(policyJson, instanceID, details.PlanID)
 	if err != nil {
-		logger.Error("get-default-policy", err)
+		logger.Error("get-default-policy: no user defined policy was provided", err)
```

Comment:
If an error is returned _something_ was provided, but it was not a valid policy, so I would not add this misleading error message. The err object should contain the actual error message and it will be logged.

```suggestion
		logger.Error("get-default-policy", err)
```